docs/src/starting-terrashine.md

```diff
@@ -18,3 +18,164 @@ Note that the `/mirror/v1/`  path is required in the URL to allow the backend se
 ```
 AWS_REGION=eu-west-1 AWS_ACCESS_KEY_ID=xxx AWS_SECRET_ACCESS_KEY=xxx RUST_LOG=info  ./terrashine  server --s3-bucket-name terrashine-example-test-1111  --http-redirect-url https://example.com/mirror/v1/
 ```
+
```

Comment:
I'd probably move this to its own "Developing terrashine" page. Keep the starting terrashine for end-users.

Reply:
yeah it makes more sense to move there. 

moved!